lib/oci_utils/migrate/image_types/vmdk.py

```diff
@@ -277,42 +265,35 @@ def type_specific_prereq_test(self):
-            _logger.critical('   Image size %8.2f GB exceeds maximum allowed %8.2f GB'
-                             % (prereqs['MAX_IMG_SIZE_GB'],
-                                self._img_info['img_size']['logical']))
+        if self.image_info['img_size']['logical'] > prereqs['MAX_IMG_SIZE_GB']:
+            _logger.critical('   Image size %8.2f GB exceeds maximum allowed %8.2f GB',
+                             prereqs['MAX_IMG_SIZE_GB'], self.image_info['img_size']['logical'])
             failmsg += '\n  Image size %8.2f GB exceeds maximum allowed ' \
                        '%8.2f GB' % (prereqs['MAX_IMG_SIZE_GB'],
-                                     self._img_info['img_size']['logical'])
+                                     self.image_info['img_size']['logical'])
             passed_requirement = False
         else:
             failmsg += '\n  Image size %8.2f GB meets maximum allowed size ' \
-                       'of %8.2f GB' % (self._img_info['img_size']['logical'],
+                       'of %8.2f GB' % (self.image_info['img_size']['logical'],
                                         prereqs['MAX_IMG_SIZE_GB'])
 
         #
-        # type:
+        # type
         if self.img_header['desc']['createType'] \
                 not in prereqs['vmdk_supported_types']:
-            _logger.critical(
-                '   Image type %s is not in the supported type list: %s' %
-                (self.img_header['desc']['createType'],
-                 prereqs['vmdk_supported_types']))
-            failmsg += '\n  Image type %s is not in the supported type list: %s' %\
-                       (self.img_header['desc']['createType'],
-                        prereqs['vmdk_supported_types'])
+            _logger.critical('   Image type %s is not in the supported type list: %s',
+                             self.img_header['desc']['createType'], prereqs['vmdk_supported_types'])
+            failmsg += '\n  Image type %s is not in the supported type list: %s' \
+                       % (self.img_header['desc']['createType'], prereqs['vmdk_supported_types'])
             passed_requirement = False
         else:
             failmsg += '\n  Image type %s is in the supported type list: %s' \
-                       % (self.img_header['desc']['createType'],
-                          prereqs['vmdk_supported_types'])
+                       % (self.img_header['desc']['createType'], prereqs['vmdk_supported_types'])
             #
             # Warning, for now, streamOptimized format will probably cause problems.
             if self.img_header['desc']['createType'] == 'streamOptimized':
-                _ = read_yn('  %s\n  Continue' % self.streamoptimized_msg,
-                            yn=False,
-                            suppose_yes=migrate_data.yes_flag)
+                _ = read_yn('  %s\n  Continue' % self.streamoptimized_msg, yn=False, suppose_yes=migrate_data.yes_flag)
```

Comment:
why are we not using the returned answer ?

Reply:
this is to get the attention of the operator of possible issues later, i.e. at instance creation and to make sure the message is read.